docs/en/05_Contributing/01_Code.md

```diff
@@ -86,6 +86,20 @@ If you are changing existing APIs, introducing new APIs or major new features,
 please review our guidance on [supported versions](release_process#supported-versions).
 You will need to change the branch for your pull request accordingly.
 
+SilverStripe public APIs explicitly include:
+ - namespaces, classes, interfaces and traits
+ - public and protected scope (including methods, properties and constants)
+ - global functions, variables
+ - yml configuration file structure
+ - private static class properties (considered to be configuration variables)
+
+SilverStripe public APIs explicitly exclude:
+ - private scope (methods and properties with the exception for `private static`)
+ - entities marked as `@internal`
+ - yml configuration file default values
```

Comment:
This feels risky to me, but I can't think of any example this would break something.

Reply:
Doesn't mean we have to change them every patch release. Simply meant to leave it for contributors' discretion